numpyro/distributions/__init__.py

```diff
@@ -55,6 +56,7 @@
     'GaussianRandomWalk',
     'HalfCauchy',
     'HalfNormal',
+    'InverseAutoRegressiveTransform',
```

Comment:
nit: I think that it is better to lower case `Regressive` to match literature. Btw, should it be `InverseAutoregressiveFlow` to match pyro name?

Reply:
ok